spec/support/bucket_list_sign_in_helper.rb

```diff
@@ -1,10 +1,14 @@
 module BucketListSignIn
   def bucket_list_sign_in
-    bucket_list = FactoryGirl.create(:bucket_list)
+    user = FactoryGirl.create(:user)
 
     visit new_user_session_path
-    fill_in 'Email', with: bucket_list.user.email
-    fill_in 'Password', with: bucket_list.user.password
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.